core/service/service.go

```diff
@@ -206,7 +212,13 @@ func (s *Service) CreateAccount(ctx context.Context, req *request.CreateAccount)
 func (s *Service) Verify(ctx context.Context, req *request.Verify) (*response.Message, error) {
 	v, err := s.db.GetEmailVerification(req.GetEmail(), req.GetHash())
 	if err != nil {
-		return nil, err
+		if strings.Contains(err.Error(), "Email or hash can not be empty") {
```

Comment:
same here

```suggestion
		if strings.Contains(err.Error(), "email or hash can not be empty") {
```

might want to check the source